src/Ormolu/Printer/Meat/Pragma.hs

```diff
@@ -16,7 +15,7 @@ import qualified Data.Text as T
 import GHC.Types.SrcLoc
 import Ormolu.Parser.CommentStream
 import Ormolu.Parser.Pragma (Pragma (..))
-import Ormolu.Printer.Combinators
+import Ormolu.Printer.Combinators hiding (Placement (..))
```

Comment:
Why should we hide placement here?

Reply:
Otherwise we get ambiguity errors:
```
src/Ormolu/Printer/Meat/Pragma.hs:85:22: error:
    Ambiguous occurrence ‘Normal’
    It could refer to
       either ‘Ormolu.Printer.Combinators.Normal’,
              imported from ‘Ormolu.Printer.Combinators’ at src/Ormolu/Printer/Meat/Pragma.hs:18:1-33
              (and originally defined
                 at src/Ormolu/Printer/Combinators.hs:321:5-10)
           or ‘Ormolu.Printer.Meat.Pragma.Normal’,
              defined at src/Ormolu/Printer/Meat/Pragma.hs:41:5
   |
85 |           Nothing -> Normal
   |                      ^^^^^^

src/Ormolu/Printer/Meat/Pragma.hs:89:20: error:
    Ambiguous occurrence ‘Normal’
    It could refer to
       either ‘Ormolu.Printer.Combinators.Normal’,
              imported from ‘Ormolu.Printer.Combinators’ at src/Ormolu/Printer/Meat/Pragma.hs:18:1-33
              (and originally defined
                 at src/Ormolu/Printer/Combinators.hs:321:5-10)
           or ‘Ormolu.Printer.Meat.Pragma.Normal’,
              defined at src/Ormolu/Printer/Meat/Pragma.hs:41:5
   |
89 |               else Normal
   |                    ^^^^^^

src/Ormolu/Printer/Meat/Pragma.hs:90:12: error:
    Ambiguous occurrence ‘Normal’
    It could refer to
       either ‘Ormolu.Printer.Combinators.Normal’,
              imported from ‘Ormolu.Printer.Combinators’ at src/Ormolu/Printer/Meat/Pragma.hs:18:1-33
              (and originally defined
                 at src/Ormolu/Printer/Combinators.hs:321:5-10)
           or ‘Ormolu.Printer.Meat.Pragma.Normal’,
              defined at src/Ormolu/Printer/Meat/Pragma.hs:41:5
   |
90 |       _ -> Normal
   |            ^^^^^^
```